terraform/azure.tf

```diff
@@ -1,5 +1,11 @@
 terraform {
   required_version = "~> 0.12.0"
+  required_providers {
```

Comment:
Are you making changes to terraform version now ?Anyway if it happens, its good since for all the changes we need to locally change the config.just checking did u do it intensionally ?